src/js/components/App.js

```diff
@@ -115,7 +115,6 @@ class App extends Component {
 
   componentDidMount() {
     window.addEventListener("keydown", this.handleKeyDown)
-    parent.postMessage({ __mocoBX: { iFrameHeight: window.document.body.scrollHeight } }, "*")
```

Comment:
I have no clue anymore why this is here or why it was necessary. It both works on Firefox and Chrome without it.